src/lib/mongoBackend/compoundValueBson.h

```diff
@@ -37,14 +37,14 @@
 *
 * compoundValueBson (for objects) -
 */
-extern void compoundValueBson(std::vector<orion::CompoundValueNode*> children, mongo::BSONObjBuilder& b);
+extern void compoundValueBson(const std::vector<orion::CompoundValueNode*>& children, mongo::BSONObjBuilder& b);
```

Comment:
It is true this is an actual change in code, not just style ...
However it is not a bug fix, *only* a fix to save space on the stack.
Do we really want to go this low level in CNR ?